tests/unittests/sources/test___init__.py

```diff
@@ -13,11 +13,13 @@
         "ci.ds=OpenStack",
         "aosiejfoij ci.ds=OpenStack blah",
         "aosiejfoij ci.ds=OpenStack faljskebflk",
+        "ci.ds=OpenStack;",
```

Comment:
Probably want the unit test extension for `ci.ds=openstack;` or something similar if we don't already have it

Reply:
+1 done